src/blocks/scratch3_motion.js

```diff
@@ -149,10 +149,10 @@ Scratch3MotionBlocks.prototype.ifOnEdgeBounce = function (args, util) {
     // and clamped to zero when the sprite is beyond.
     var stageWidth = this.runtime.constructor.STAGE_WIDTH;
     var stageHeight = this.runtime.constructor.STAGE_HEIGHT;
-    var distLeft = Math.max(0, stageWidth / 2 + bounds.left);
-    var distTop = Math.max(0, stageHeight / 2 - bounds.top);
-    var distRight = Math.max(0, stageWidth / 2 - bounds.right);
-    var distBottom = Math.max(0, stageHeight / 2 + bounds.bottom);
+    var distLeft = Math.max(0, (stageWidth / 2) + bounds.left);
+    var distTop = Math.max(0, (stageHeight / 2) - bounds.top);
+    var distRight = Math.max(0, (stageWidth / 2) - bounds.right);
+    var distBottom = Math.max(0, (stageHeight / 2) + bounds.bottom);
```

Comment:
I'm not 100% sure about this one given, "Please Excuse My Dear Aunt Sally" but it does make things a little easier to read so 👍